rascal-vscode-extension/src/lsp/RascalLSPConnection.ts

```diff
@@ -63,10 +63,12 @@ export async function activateLanguageClient(
 
     schemesReply.then( schemes => {
         vfsServer.ignoreSchemes(schemes);
-        new RascalFileSystemProvider(client).registerSchemes(schemes);
+        const allRegistered = new RascalFileSystemProvider(client).tryRegisterSchemes(schemes);
+        if (!allRegistered) {
+            client.warn(`At least one of the expected schemes failed to register.\nExpected schemes: ${schemes}`);
```

Comment:
not sure if this should be a warning, as we expect races? So sometimes it happens, and that's fine?